src/ResourceManager/Network/Commands.Network/VirtualNetworkGatewayConnection/NewAzureRmIpsecPolicyCommand.cs

```diff
@@ -22,15 +22,15 @@ namespace Microsoft.Azure.Commands.Network
     public class NewAzureRmIpsecPolicyCommand : NetworkBaseCmdlet
     {
         [Parameter(
-            Mandatory = true,
+            Mandatory = false,
             HelpMessage = "The IPSec Security Association (also called Quick Mode or Phase 2 SA) lifetime in seconds")]
-        [ValidateNotNullOrEmpty]
+        [ValidateRange(300, 172799)]
         public int SALifeTimeSeconds { get; set; }
 
         [Parameter(
-            Mandatory = true,
+            Mandatory = false,
             HelpMessage = "The IPSec Security Association (also called Quick Mode or Phase 2 SA) payload size in KB")]
-        [ValidateNotNullOrEmpty]
+        [ValidateRange(1024, int.MaxValue)]
```

Comment:
@henry416 adding these `ValidateRange` attributes are breaking changes. Previously if a user provided a value outside of the above range, what would happen?